spec/features/guides_spec.rb

```diff
@@ -6,22 +6,23 @@
   let (:crop) { FactoryGirl.create(:crop, name: 'radish') }
   let (:user) { FactoryGirl.create(:user) }
 
-  # TODO: Figure out why these don't work.
-  # it 'shows individual guides', js: true do
-  #   visit "guides/#{guide._id}"
-  #   expect(page).to have_content('Test Guide')
-  # end
+  it 'shows individual guides', js: true do
+    visit guide_path(id: guide._id)
+    expect(page).to have_content('Test Guide')
+  end
 
-  # it 'shows the create guide page', js: true do
-  #   login_as user
-  #   visit new_guide_path
-
-  #   sleep 5
-  #   # puts "html: #{page.html}"
-  #   binding.pry
-  #   expect(page).to have_content('Lets Get Started')
-  #   expect(page).to have_button('Choose a crop to continue')
-  #   fill_in :crop_name, with: crop.name
-  #   expect(page).to have_link('radish')
-  # end
+  it 'shows the create guide page', js: true do
+    login_as user
+    visit new_guide_path
+    expect(page).to have_content('Lets Get Started!')
+    # Capybarra doesn't search the value of disabled buttons???
+    commit_button = page.find('input[name="commit"]')
+    expect(commit_button["value"]).to have_content('Choose a crop to continue')
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.